tests/unit/fem/test_build_dof_to_arrays.cpp

```diff
@@ -110,8 +110,6 @@ TEST_CASE("Build Dof To Arrays",
             FiniteElementSpace fespace(mesh, fec);
             int size = fespace.GetTrueVSize();
 
-            fespace.BuildDofToArrays();
-
             for (int i = 0; i<size; i++)
```

Comment:
Not super important, but if you have such a nice test, how about just adding here a loop over boundaries to check the new method? 😉

Reply:
@najlkin Apologies for the late reply - I have not been available to work on this.

I could but I think I would also have to update the `GetMesh` function at the bottom of this file to include boundary elements for all the different meshes which will take a good deal of time - it does not seem that boundary elements are automatically included?

Reply:
It is easy, just add `mesh->GenerateBoundaryElements()` at the end of `GetMesh()` 😉 .